plugins/woocommerce/tests/e2e-pw/daily.playwright.config.js

```diff
@@ -0,0 +1,8 @@
+const defaultConfig = require( './playwright.config' );
+
+const config = {
+	...defaultConfig,
+	testIgnore: '**/shopper/**',
+};
+
+module.exports = config;
```

Comment:
I added a separate Playwright config file for the daily tests so that it's easier to disable the shopper tests for the meantime